lib/hound/helpers.ex

```diff
@@ -21,11 +21,13 @@ defmodule Hound.Helpers do
   end
 
 
-  defmacro hound_session do
+  defmacro hound_session(opts \\ []) do
     quote do
       setup do
-        Hound.start_session
-        on_exit fn-> Hound.end_session end
+        pid = Hound.start_session(unquote(opts))
+        on_exit fn ->
+          Hound.end_session(pid)
```

Comment:
This callback should not be required as start_session should monitor the caller and shut it down.